apps/examples/st_things/light/sample_light_user.c

```diff
@@ -26,6 +26,10 @@
 
 #define TAG	"LIGHT_USER"
 
+#define PUSH_MESG_SOURCE_ATTRIBUTE		"x.org.iotivity.ns.source"
```

Comment:
Could you please use the spaces instead of tabs.

Reply:
fixed.